examples/python/route_guide/run_codegen.py

```diff
@@ -28,5 +27,16 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Runs the protoc with gRPC plugin to generate protocol messages and gRPC stubs.
-python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/route_guide.proto
```

Comment:
I didn't realize this was part of a larger demo change.  To me, the new quick-start feels like it got more complicated, especially the explicit use of Python2.7, which would imply that grpcio or grpcio-tools doesn't work with Python3, which shouldn't be the case.

I'll defer my opinion, but I still see a reference to 
```python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/route_guide.proto```

in the aforementioned PR which is no longer accurate after this change.